src/Output/Copy/Schema/Mapper/SchemaMapperInterface.php

```diff
@@ -7,8 +7,6 @@
 interface SchemaMapperInterface
 {
     /**
-     * @param array $payload
-     *
      * @return SerializableInterface
      */
     public function toSchema(array $payload);
```

Comment:
we can't because child classes type hint this differently already :( with the actual Mapped schema, e.g.:
```php
class StoryMapper implements SchemaMapperInterface
{
...
    /**
     * @param array $payload
     *
     * @throws UnexpectedResponseBodyException
     *
     * @return Story
     */
    public function toSchema(array $payload): Story
```